sdk/parents/azure-data-sdk-parent/pom.xml

```diff
@@ -441,6 +443,18 @@
           </reportSet>
         </reportSets>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+        <configuration>
+          <environmentVariables>
+            <test.mode>${testMode}</test.mode>
+            <KEYVAULT_VAULTURI>https://azure-keyvault-3.vault.azure.net</KEYVAULT_VAULTURI>
+            <KEYVAULT_VAULTURI_ALT>https://azure-keyvault-2.vault.azure.net</KEYVAULT_VAULTURI_ALT>
+          </environmentVariables>
```

Comment:
The parent pom had these two additional lines. Those should be copied to this pom as well. 

```
<configurationAPIEnvironmentFound>environmentConfiguration</configurationAPIEnvironmentFound>	
<configurationAPIUseRuntimeFirst>environmentConfiguration</configurationAPIUseRuntimeFirst>
```

Reply:
These were being used in tests for Azure Core, will update those tests to no longer require these configurations.